Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.Properties.cs

```diff
@@ -0,0 +1,214 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// TabView properties.
+    /// </summary>
+    public partial class TabView
+    {
+        /// <summary>
+        /// Gets or sets the content to appear to the left or above the tab strip.
+        /// </summary>
+        public object TabStartHeader
```

Comment:
UWP unified that controls have a built-in header / footer for [form labels](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/labels), since ListView already defines these in a similar fashion, I'm keeping that behavior the same.

Reply:
Header and Footer do not need to be placed at the top and bottom. Even UWP is now allowing headers for elements to be to the left instead of top.

Reply:
There isn't a need for two header properties and two footer properties. One for each is enough

Reply:
Actually this is just a more flexible way for devs to customize the look and feel of the control. It has a header on the top and a footer on the bottom; a start header which is on the same row of the tab item headers and aligned to the left; an action header which is placed right next to the last tab item header; an end header which is aligned to the far right.

Reply:
I understand why they were added. My point is they are not needed. The tabs are essentially a horizontal ListView. In a horizontal list, the header and foot _should_ appear to the left and right of the items. Same for this. There is no need to have a header and footer for the top and bottom when those can be easily added. the action area can easily be combined into one and allow the developer to decide if they want all items on the left, all on the right, or to separate items.

Reply:
Not really. Although the tab headers go from left to right by default, the _flow_ of the control is still from top to bottom 'cause its main focus element is still the content area. Given it's derived from the `ListView` already, it just makes sense to align with the design. For other modes, we can adjust the position of the headers in future releases too.